frontend/src/components/VContentLink/meta/VContentLink.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs"
+
+import VContentLink from "~/components/VContentLink/VContentLink.vue"
+
+<Meta title="Components/VContentLink" components={VContentLink} argTypes={{}} />
+
+export const contentLinkArgTypes = {
+  mediaType: { options: ["audio", "image"], control: { type: "radio" } },
+  resultsCount: { control: { type: "number" } },
+  isSelected: { control: { type: "boolean" } },
+  layout: { options: ["stacked", "horizontal"], control: { type: "radio" } },
+}
+
+export const VContentLinkStory = (args) => ({
+  template: `<VContentLink v-bind="args" />`,
+  components: { VContentLink },
+  setup() {
+    return { args }
+  },
+})
+
+export const VContentLinkHorizontalStory = (args) => ({
+  template: `<div class="max-w-md"><VContentLink v-bind="args" /></div>`,
+  components: { VContentLink },
+  setup() {
+    return { args }
+  },
+})
+
+export const TwoVContentLinkStory = (agrs) => ({
+  template: `
+  <div class="grid grid-cols-2 gap-4">
+    <VContentLink
+      v-for="(type, key) in types"
+      :key="key"
+      :media-type="type.mediaType"
+      :results-count="type.resultsCount"
+    />
+  </div>
+  `,
+  components: { VContentLink },
+  data() {
+    return {
+      types: [
+        { mediaType: "image", resultsCount: 4321 },
+        { mediaType: "audio", resultsCount: 1234 },
+      ],
+    }
+  },
+  methods: {
+    onSelected(mediaType) {
+      this.selected = mediaType
+    },
+  },
+})
+
+# VContentLink
+
+<Description of={VContentLink} />
+
+<ArgsTable of={VContentLink} />
+
+## Default VContentLink
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      mediaType: "image",
+      resultsCount: 5708,
+      isSelected: false,
+    }}
+    argTypes={contentLinkArgTypes}
+  >
+    {VContentLinkStory.bind({})}
+  </Story>
+</Canvas>
+
+## Horizontal VContentLink
+
+<Canvas>
+  <Story
+    name="Horizontal"
+    args={{
+      mediaType: "audio",
+      resultsCount: 4561,
+      isSelected: false,
+      layout: "horizontal",
+    }}
+    argTypes={contentLinkArgTypes}
+  >
+    {VContentLinkHorizontalStory.bind({})}
+  </Story>
+</Canvas>
+
+## Mobile VContentLink
+
```

Comment:
Please include a note that this story should be viewed in the Canvas tab. I was a bit confused when the "mobile" story was same as the desktop one when seen in the Docs tab.